pimd/pim6_cmd.c

```diff
@@ -448,6 +448,65 @@ DEFPY (no_ipv6_pim_rp_prefix_list,
 	return pim_process_no_rp_plist_cmd(vty, rp_str, plist);
 }
 
+
+DEFPY (ipv6_ssmpingd,
+       ipv6_ssmpingd_cmd,
+       "ipv6 ssmpingd [X:X::X:X]$source",
+       IPV6_STR
+       "Enable ssmpingd operation\n"
+       "Source address\n")
+{
+	int idx_ipv4 = 2;
+        const char *src_str = (argc == 3) ? argv[idx_ipv4]->arg : "::";
+        const char *vrfname;
```

Comment:
Please introduce a common function for IPV4 and IPV6 in pim_common.c and move these code to there.
We are already doing for other clis. Please refer https://github.com/FRRouting/frr/pull/10393

Reply:
Ack